### PR TITLE
[artifactory-ha] Support list of custom secrets

### PR DESCRIPTION
**Pull request by anastasiagrinman**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -858,6 +858,20 @@ artifactory:
     ## Custom volume comes here ##
 ```
 
+### Custom secrets
+If you need to add a custom secret in a custom init or sidecar container, you can use this option.
+
+For this, there is a section for defining custom secrets in the [values.yaml](values.yaml). By default it's commented out
+```yaml
+artifactory:
```
Comment (chukka) [inline stable/artifactory-ha/README.md:866]: @anastasiagrinman Can you please add this parameter in Configuration section in README  - https://github.com/jfrog/charts/blob/master/stable/artifactory/README.md#configuration

Comment (anastasiagrinman) [reply stable/artifactory-ha/README.md:866]: Done.

Comment (anastasiagrinman) [reply stable/artifactory-ha/README.md:866]: artifactory chart will be provided as different PR.

File: stable/artifactory-ha/README.md [hunk 2]
```diff
@@ -858,6 +858,20 @@ artifactory:
...
+  # Add custom secrets - secret per file
+    customSecrets:
+      - name: custom-secret
+        key: custom-secret.yaml
+        config: >
+          configuration file content
```
[the remark this answers is not in the document]
Comment (anastasiagrinman) [reply stable/artifactory-ha/README.md]: Done.

File: stable/artifactory-ha/README.md [hunk 3]
```diff
@@ -858,6 +858,20 @@ artifactory:
...
+If you need to add a custom secret in a custom init or sidecar container, you can use this option.
+
+For this, there is a section for defining custom secrets in the [values.yaml](values.yaml). By default it's commented out
+```yaml
+artifactory:
+  # Add custom secrets - secret per file
+    customSecrets:
+      - name: custom-secret
+        key: custom-secret.yaml
+        config: >
```
[the remark this answers is not in the document]
Comment (anastasiagrinman) [reply stable/artifactory-ha/README.md]: Done.

**Comment by chukka** [comment]
@anastasiagrinman Can you please rebase and update PR

**Pull request merged**